price_parser/parser.py

```diff
@@ -36,15 +37,36 @@ def fromstring(cls, price: Optional[str],
         which may contain currency, as a hint. If currency is present in
         ``price`` string, it could be **preferred** over a value extracted
         from ``currency_hint`` string.
+
+        ``thousand_with_dot`` is optional; it is used to determine if
+        thousand separator is a dot or a comma. If ``thousand_with_dot``
+        is ``True``, then ``1.000`` is parsed as ``1000``. If it is ``False``,
+        then ``1.000`` is parsed as ``1``.
         """
+        currency = extract_currency_symbol(price, currency_hint)
+        if currency is not None:
+            currency = currency.strip()
+        for currency_key in CURRENCIES.keys():
+            if CURRENCIES[currency_key]["s"] == currency:
+                if CURRENCIES[currency_key]["d"] == 3:
+                    thousand_with_dot = False
+                break
+            if CURRENCIES[currency_key]["sn"] == currency:
+                if CURRENCIES[currency_key]["d"] == 3:
+                    thousand_with_dot = False
+                break
+            alternative_symbols = CURRENCIES[currency_key].get("sn2", [])
+            if currency in alternative_symbols:
+                if CURRENCIES[currency_key]["d"] == 3:
+                    thousand_with_dot = False
+                break
```

Comment:
Could you see if you can create some new variable in `_currency`, like [the existing ones at the end](https://github.com/scrapinghub/price-parser/blob/18c5ed8c98a569f805ac4444a2dae20bdef27760/price_parser/_currencies.py#L1730-L1736), to avoid looping here?